dna/src/lib.rs

```diff
@@ -165,25 +165,24 @@ impl Dna {
         zome: &'a zome::Zome,
         capability_name: &str,
     ) -> Option<&'a wasm::DnaWasm> {
-        zome.capabilities
+        let capability = zome
+            .capabilities
             .iter()
-            .find(|c| c.name == capability_name)
-            .map(|capability| &capability.code)
+            .find(|c| c.name == capability_name)?;
+        Some(&capability.code)
```

Comment:
I believe calling `map` on an `Option<T>` is more idiomatic.

Reply:
It seems to me that `get_wasm_for_capability`
https://github.com/holochain/holochain-rust/blob/587f2335daf028441e1de8c45ee127b753cbdc9c/dna/src/lib.rs#L176-L186
and `get_validation_bytecode_for_entry_type`
https://github.com/holochain/holochain-rust/blob/587f2335daf028441e1de8c45ee127b753cbdc9c/dna/src/lib.rs#L189-L201
are perfect cases to use the `?` operator on `Option`s. Given the context of that these are the two functions next to `get_capability`, it seems to make sense to me to continue the `?` pattern. If you disagree I can change these to `and_then`s and `map`s.
https://github.com/holochain/holochain-rust/blob/587f2335daf028441e1de8c45ee127b753cbdc9c/dna/src/lib.rs#L162-L201

Reply:
No, sure leave it